itest/lib/app.js

```diff
@@ -23,18 +22,21 @@ const appStep = async (stepMessage, f) => {
 
 export const newAppInstance = (name: string, idx: number): Application => {
   const macInstallPath = "/Applications/Brim.app/Contents/MacOS/Brim"
+  const userDataDir = path.resolve(path.join(LOGDIR, name, idx.toString()))
+  mkdirpSync(userDataDir)
 
   // https://github.com/electron-userland/spectron#new-applicationoptions
   let appArgs = {
     // Some version of Spectron choose a random debugging port in the 9000-9999
     // range. Since zqd uses 9867, set this to 9999.
-    chromeDriverArgs: ["remote-debugging-port=9999"],
+    chromeDriverArgs: [
+      "--remote-debugging-port=9999",
```

Comment:
You'll want to make sure this line gets removed as part of any conflict resolution. Ref https://github.com/brimsec/brim/pull/628